custom_components/hubitat/binary_sensor.py

```diff
@@ -57,7 +58,7 @@ def is_on(self) -> bool:
     @property
     def name(self) -> str:
         """Return the display name for this sensor."""
-        return f"{super().name} {self._attribute}"
```

Comment:
Instead of adding a separate method, why not just do

```python
return f"{super().name} {self._attribute}".title()
```

Reply:
Didn't realize there was a `title()` method, sounds reasonable.